modules/tests/go.mod

```diff
@@ -124,15 +124,18 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
+// Kubernetes
 replace (
 	k8s.io/api => k8s.io/api v0.23.4
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.4
 	k8s.io/apimachinery => k8s.io/apimachinery v0.23.4
 	k8s.io/client-go => k8s.io/client-go v0.23.4
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.4
 )
 
+replace github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
```

Comment:
this is also required by CDI https://github.com/kubevirt/containerized-data-importer/blob/main/go.mod#L150. Can you please move this to the same block and maybe put a `// CDI ` header for navigability :) ?